include/service/entities/site_service.js

```diff
@@ -0,0 +1,376 @@
+var async = require('async');
+var url = require('url');
+var util  = require('../../util.js');
+
+module.exports = function SiteServiceModule(pb) {
+
+
+    /**
+     * Service for performing site specific operations.
+     *
+     * @module Services
+     * @submodule Entities
+     * @class SiteService
+     * @constructor
+     */
+    function SiteService(){}
```

Comment:
The site service is a huge part of the changes that are going in.  I'd like to get code level documentation on all of the properties and functions.